packages/schemas/src/foundations/jsonb-types.ts

```diff
@@ -44,13 +52,83 @@ export const userLogPayloadGuard = z.object({
 
 export type UserLogPayload = z.infer<typeof userLogPayloadGuard>;
 
+/**
+ * Connectors
+ */
+
 // TODO: support empty shape of object
 export const connectorConfigGuard = z.object({});
 
 export type ConnectorConfig = z.infer<typeof connectorConfigGuard>;
 
+/**
+ * Settings
+ */
+
 export const adminConsoleConfigGuard = z.object({
   applicationSkipGetStarted: z.boolean(),
 });
 
 export type AdminConsoleConfig = z.infer<typeof adminConsoleConfigGuard>;
+
+/**
+ * SignIn Experience
+ */
+
+export const companyInfoGuard = z.object({
+  name: z.string(),
+  logo: z.string(),
+});
+
+export type CompanyInfo = z.infer<typeof companyInfoGuard>;
+
+export enum BrandingStyle {
+  CompanyLogo_CompanyName_AppName = 'CompanyLogo_CompanyName_AppName',
+  CompanyLogo_AppLogo_CompanyName_AppName = 'CompanyLogo_AppLogo_CompanyName_AppName',
+  AppLogo_CompanyName_AppName = 'AppLogo_CompanyName_AppName',
+}
+
+export const brandingGuard = z.object({
+  primaryColor: z.string(),
+  backgroundColor: z.string(),
+  darkMode: z.boolean(),
+  darkPrimaryColor: z.string(),
+  darkBackgroundColor: z.string(),
+  style: z.nativeEnum(BrandingStyle),
+});
+
+export type Branding = z.infer<typeof brandingGuard>;
+
+export const termsOfUseGuard = z.object({
+  enabled: z.boolean(),
+  content: z.string().optional(),
+  contentUrl: z.string().optional(),
+});
+
+export type TermsOfUse = z.infer<typeof termsOfUseGuard>;
+
+export enum Language {
+  english = 'en',
+  chinese = 'zh-cn',
+}
+
+export const localizationGuard = z.object({
+  autoDetect: z.boolean(),
+  primaryLanguage: z.nativeEnum(Language),
+  fallbackLanguage: z.nativeEnum(Language),
+});
+
+export type Localization = z.infer<typeof localizationGuard>;
+
+export const signInMethodSettingsGuard = z.object({
+  primary: z.string().array().nonempty().max(3),
+  secondary: z.string().array(),
+  disabled: z.string().array(),
+});
+
+export type SignInMethodSettings = z.infer<typeof signInMethodSettingsGuard>;
+
+// TODO: Leave it for now
+export const signInMethodMetadataGuard = z.object({});
```

Comment:
should we just remove this until needed? or we can keep it if we're certain to use in the near future.

Reply:
updated